pkg/apply/solver/solver_test.go

```diff
@@ -183,7 +183,7 @@ func TestTaskQueueBuilder_AppendApplyWaitTasks(t *testing.T) {
 						testutil.ToIdentifier(t, resources["deployment"]),
 						testutil.ToIdentifier(t, resources["secret"]),
 					},
-					taskrunner.AllCurrent, 1*time.Second,
+					taskrunner.AllCurrent, 1*time.Minute,
```

Comment:
Does it matter what the timeout is for this test? We're just checking that it was set, right?

Reply:
Before this PR, we didn't check the Timeout field at all.